app/assets/src/components/visualizations/heatmap/Heatmap.js

```diff
@@ -968,7 +968,17 @@ export default class Heatmap {
       let columnMetadataCellEnter = columnMetadataCell
         .enter()
         .append("rect")
-        .attr("class", "columnMetadataCell");
+        .attr("class", "columnMetadataCell")
+        .on("mouseover", d => {
+          this.options.onNodeHover && this.options.onNodeHover(d);
```

Comment:
You can also leverage/replicate part of the code in `getColumnMetadataLegend`, which returns the aggregated metadata information.
So, i guess you could just return the label here.

Reply:
good idea, thanks